tests/toolkit/matl_sell_policy_tests.cc

```diff
@@ -100,9 +100,23 @@ TEST_F(MatlSellPolicyTests, Bids) {
   ASSERT_EQ(obs.size(), 1);
   ASSERT_EQ((*obs.begin())->bids().size(), 2);
   ASSERT_FLOAT_EQ((*(*obs.begin())->bids().begin())->offer()->quantity(),
-                  mat->quantity() / 2);
+                  qty / 2);
+  ASSERT_EQ((*(*obs.begin())->bids().begin())->offer()->comp(), comp1);
+
+  // If available quantity is bigger than the quanta only an integer number of quanta is offered
+  p.Init(NULL, &buff, "", qty, true, 2).Set(commod);
```

Comment:
Should we use a variable for the final argument, and then reuse that variable in line 112?  What is important about this variable vs the previous test?  That it's an integer? That it's smaller than `qty` does it matter how much smaller?

Reply:
maybe one need to clarify the initial condition for this (all?) problem(s)